android_tutorial_cv_bridge_CompressedImage_javacv/src/main/java/com/github/rosjava/android_extras/android_tutorial_cv_bridge/MainActivityCompressedJavacv.java

```diff
@@ -0,0 +1,148 @@
+package com.github.rosjava.android_extras.android_tutorial_cv_bridge;
```

Comment:
I remove licences from source files that I write. 
Already I contribute to rosjava_extra like that and as we discuss here:
https://github.com/rosjava/rosjava_extras/pull/15#issuecomment-274533178